scripts/docker/Dockerfile.openmptarget

```diff
@@ -1,4 +1,4 @@
-ARG BASE=nvidia/cuda:10.1-devel
+ARG BASE=nvidia/cuda:11.1-devel-ubuntu20.04
```

Comment:
Is this necessary?

Reply:
I don't know.  I wanted an updated GCC toolchain which I get with ubuntu 20.04